### DISCUSSION TITLE
V3 Docs there is nothing show the user how to build or customize his own page layout

### DISCUSSION THREAD

**Discussion by devhoussam1998** [filamentphp/filament#9461] [Help]
the filament core team should take in considiration to explain or make the build or custimization easy for filament page layout.

**Comment by VincvanAs** [comment]
First of all, this is an open-source project. If you believe something is missing, please don your work gloves and contribute. Secondly, there is a wealth of information available on extending or overriding layouts in both the documentation and the discussions area
